airmozilla/main/templates/main/home.html

```diff
@@ -11,7 +11,16 @@
 </style>
 {% endblock %}
 
-
+{% block extra_head %}
+{{ super() }}
+<meta property="og:title" content="{{ page_title }}">
+<meta property="og:url" content="{{ request.build_absolute_uri() }}">
+<meta property="og:description" content="Air Mozilla is the Internet multimedia presence of Mozilla, with live and pre-recorded shows, interviews, news snippets, tutorial videos, and features about the Mozilla community.">
+{% set thumb = thumbnail('img/logo-airmo-md.png', '200x200', crop='center') %}
```

Comment:
This doesn't work. It's not easy to know but you can't generate thumbnails from static images. Only for images that are uploaded as part of the data. 
Instead I would just opt to finding out the width and height is of that logo and typing that in. E.g. `<meta property="og:image:width" content="222">`

And for the `og:image` you need to use a function that returns a static URL that is absolute. I think a good example of this is here: https://github.com/mozilla/airmozilla/blob/master/airmozilla/roku/templates/roku/categories.xml
`abs_static`.